swarm_copy/stream.py

```diff
@@ -21,7 +21,7 @@ async def stream_agent_response(
 ) -> AsyncIterator[str]:
     """Redefine fastAPI connections to enable streaming."""
     if isinstance(agents_routine.client, AsyncOpenAI):
-        connected_agents_routine = AgentsRoutine(
+        connected_agents_routine = AgentsRoutine(  # type: ignore[call-arg]
```

Comment:
I did not implement the logic here. I mean nothing would change really...just did not want to overcomplicate this poc